cmd/utils/cmd.go

```diff
@@ -40,7 +41,9 @@ import (
 )
 
 const (
-	importBatchSize = 2500
+	importBatchSize       = 2500
+	freeDiskSpaceWarning  = 1024 * 1024 * 1024
+	freeDiskSpaceCritical = 300 * 1024 * 1024
```

Comment:
I suggest setting the warning-level to a multiple of the critical level, maybe 
```
freeDiskSpaceWarning  = 2 * freeDiskSpaceCritical
```
But with such a change, we could just inline it and remove the `freeDiskSpaceWarning` constant...

Reply:
fixed